embark/lib/templates/.env

```diff
@@ -0,0 +1,7 @@
+# UNCOMENT THE LINE WITH THE ENVIRONMMENT VARIABLE YOU WANT TO OVERRIDE THE DEFAULT VALUE IN DOCKER COMPOSE
```

Comment:
How does this work if you have more than one service defined in docker-compose

Reply:
If you look the variables in this env file matches the ones defined with default values in the docker compose file.

So when you uncoment  here a var it will override all occurrences of it in the docker compose file.

A more complex example of a docker compose file can be found [here](https://gitlab.com/exadra37-docker-compose/php7/docker-stack/blob/final/src/docker-compose.yml).